packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -163,8 +163,12 @@ class DebuggerScreen extends Screen {
                         ])
                         ..small(),
                     ]),
-                  div()..flex(),
                   breakOnExceptionControl,
+                  div()..flex(),
+                  div(c: 'btn-group margin-left')
+                    ..add([
+                      createTipsButton('Debugger', 'debugger'),
```

Comment:
all these cases the two fields are the same modulo camel case. 
Go ahead and just pass the one name and lower case to generate the url.

Reply:
done.